virtualhost.sh

```diff
@@ -112,7 +161,7 @@ if [ "$action" == 'create' ]
 		a2ensite $domain
 
 		### restart Apache
-		/etc/init.d/apache2 reload
+		/etc/init.d/apache2 restart
```

Comment:
Exactly, restart includes reload, I think, but to avoid a full restart, reload action is enough.